drivers/wireless/ieee80211/bcm43xxx/bcmf_sdio.c

```diff
@@ -774,11 +769,13 @@ int bcmf_bus_sdio_initialize(FAR struct bcmf_dev_s *priv,
 
   /* Spawn bcmf daemon thread */
 
+  snprintf(arg1, 16, "0x%" PRIxPTR, (uintptr_t)priv);
```

Comment:
I think `%p` should also give the same result

Reply:
Done